announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/tool/AnnouncementAction.java

```diff
@@ -1426,16 +1426,7 @@ private boolean isOkayToDisplayMessageMenu(AnnouncementActionState state)
 	 */
 	private boolean isOkToShowOptionsButton(String statusName)
 	{
-		if (SiteService.allowUpdateSite(ToolManager.getCurrentPlacement().getContext()) && !isOnWorkspaceTab())
-		{
-			return (statusName == null || statusName.equals(CANCEL_STATUS) || statusName.equals(POST_STATUS)
-					|| statusName.equals(DELETE_ANNOUNCEMENT_STATUS) || statusName.equals(FINISH_DELETING_STATUS)
-					|| statusName.equals("noSelectedForDeletion") || statusName.equals(NOT_SELECTED_FOR_REVISE_STATUS));
-		}
-		else
-		{
-			return false;
-		}
+		return SiteService.allowUpdateSite(ToolManager.getCurrentPlacement().getContext()) && !isOnWorkspaceTab();
```

Comment:
Can you briefly explain why the 10 lines of code above can be consolidated down to one line?